dali/operators/signal/fft/spectrogram.cc

```diff
@@ -87,15 +88,18 @@ is padded with zeros.
 )",
     TensorLayout("ft"));
 
+template <bool time_major>
```

Comment:
Please update `layout` docs and remove GPU backend specific info.

Reply:
done